lib/cim/prefix-iri-provider.ts

```diff
@@ -0,0 +1,18 @@
+import {IriProvider} from "./iri-provider";
+
+const prefix = "pim:";
```

Comment:
I would use capital letters for constants, it is also not clear how was this constant selected and if the value is arbitrary and can be changed without breaking somethin.